lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -344,21 +344,39 @@ mixin _WheelEventListenerMixin on _BaseAdapter {
     }
 
     final List<ui.PointerData> data = <ui.PointerData>[];
-    _pointerDataConverter.convert(
-      data,
-      change: ui.PointerChange.hover,
-      timeStamp: _BaseAdapter._eventTimeStampToDuration(event.timeStamp!),
-      kind: ui.PointerDeviceKind.mouse,
-      signalKind: ui.PointerSignalKind.scroll,
-      device: _mouseDeviceId,
-      physicalX: event.clientX.toDouble() * ui.window.devicePixelRatio,
-      physicalY: event.clientY.toDouble() * ui.window.devicePixelRatio,
-      buttons: event.buttons!,
-      pressure: 1.0,
-      pressureMax: 1.0,
-      scrollDeltaX: deltaX,
-      scrollDeltaY: deltaY,
-    );
+    if (event.ctrlKey) {
```

Comment:
What happens on Mac with ctrl and cmd?

Reply:
ctrlKey is just how the browser sends scroll events, for legacy reasons. It's still called ctrlKey on the mac. In my mac Chrome, neither seem to cause a zoom if i hold them and scroll.

Reply:
Wheel events with `ctrlKey` on Mac do not zoom/scale. Instead they just scroll. But with this PR, the scrolling will stop, right?

Reply:
My understanding is that ctrl (not cmd) and scroll will continue to do the same thing (scroll) after this PR, is that right @moffatman?